pkg/docs/appliance.go

```diff
@@ -285,4 +285,48 @@ NOTE: Although the '--include' and '--exclude' flags are provided as options her
 			},
 		},
 	}
+
+	ApplianceSeedDocs = CommandDoc{
+		Short: "Export seed for an inactive Appliance",
+		Long: `Generate a seed file in JSON (or iso format)
+
+--validity-days
+    Seed Lifetime
+    Seeds contain temporary keys which allow access to the Controllers. The seed lifetime should therefore be kept to a minimum.
+
+--allow-customization
+    Allow appliance customizations
+    Appliance customizations are disabled by default. These can be enabled later using a cz-config command if required.
+
+SSH Authentication Method:
+    --provide-cloud-ssh-key
+    Use SSH key provided by the cloud instance
+    With Cloud based appliances, you should specify a key when launching the instance.
+
+    --ssh-key
+    Use SSH public key
+    Begins with ssh-rsa, ssh-dss, ssh-ed25519, ecdsa-sha2-nistp256, ecdsa-sha2-nistp384 or ecdsa-sha2-nistp521.
```

Comment:
Help for the flags are already in the flag description provided by the command function. This is duplicate information.